src/util/Text.ts

```diff
@@ -2,8 +2,35 @@
 // so we replace them with non-breaking-space characters
 export function indentWithNBS(str: string):  string {
     const fakeTab = "\u00A0".repeat(4);
-    
+
     str = str.replace(/\t/g, fakeTab);
     str = str.replace(/    /g, fakeTab);
     return str;
+}
+
+export function stripIndent(strings: TemplateStringsArray, ...placeholders: string[]): string {
```

Comment:
please add a comment explaining how this gets called with the template literals, since it won't look like a normal fuction call `<3`